srcpkgs/man-db/template

```diff
@@ -19,8 +19,8 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://www.nongnu.org/man-db/"
 changelog="https://git.savannah.gnu.org/cgit/man-db.git/plain/NEWS"
-distfiles="${NONGNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
-checksum=7b383f26c67525205f96a4a3bfb27993396992794adda6fe05ad565f369f047c
+distfiles="${NONGNU_SITE}/man-db/man-db-${version}.tar.xz"
```

Comment:
any reason to change this?

Reply:
I had almost the same initial reaction when a core team member directed me to replace ${pkgname} with the actual name in the homepage. Functionally they are the same but having a variable when something does not vary and it the variable itself is not shorter than the value seems wrong and makes the result a little less readable.